src/rpc/blockchain.cpp

```diff
@@ -913,6 +913,151 @@ static UniValue getblock(const JSONRPCRequest& request)
     return blockToJSON(block, tip, pblockindex, verbosity >= 2);
 }
 
+static UniValue listprunelocks(const JSONRPCRequest& request)
+{
+    RPCHelpMan{"listprunelocks",
+        "\nReturns a list of pruning locks.\n",
+        {},
+        RPCResult{
+            RPCResult::Type::OBJ, "", "",
+            {
+                {RPCResult::Type::ARR, "prune_locks", "",
+                {
+                    {RPCResult::Type::OBJ, "", "",
+                    {
+                        {RPCResult::Type::STR, "id", "A unique identifier for the lock"},
+                        {RPCResult::Type::STR, "desc", "A description of the lock's purpose"},
+                        {RPCResult::Type::STR, "height", "A description of the lock's purpose"},
```

Comment:
This line should be removed I think (duplicate "height" with wrong description).

Reply:
Fixed